File: java/URI_1064.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1064 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double sum = 0;
		int totpositivo = 0;
		
		for(int i = 0; i < 6; i++) {
			double n = sc.nextDouble();
			if(n > 0) {
				totpositivo += 1;
				sum += n;
			}			
		}
		System.out.println(totpositivo + " valores positivos");
		double media = sum / totpositivo;
		System.out.printf("%.1f%n", media);
		
		sc.close();

	}

}

File: kotlin/URI_1064.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    var soma = 0.0
    var positivos = 0
    for (i in 1..6) {
        val n = sc.nextDouble()
        if (n > 0) {
            soma += n
            positivos++
        }
    }
    println("$positivos valores positivos")
    println("%.1f".format(soma/positivos))
}
